developer/src/server/src/version-data.ts

```diff
@@ -42,6 +44,7 @@ export function extractVersionData(version: string): Environment {
     versionTag: versionData[4],
     versionWithTag: version,
     versionEnvironment: versionData[6] || versionData[5] || 'stable',
+    versionGitTag: 'release@'+version,
```

Comment:
From the comment in l.15, should this be `+version` or `+versionWithTag`?

Reply:
See line 45 -- `version` is equivalent to `versionWithTag` :grin: